enums.h

```diff
@@ -1835,6 +1841,12 @@ enum text_color {
     COL_GOLD  = 0x3,
 };
 
+enum item_color {
+    ICOL_WHITE = 197,
+    ICOL_BLUE  = 181,
+    ICOL_RED   = 229,
+};
+

```

Comment:
Should these be based on the global palette defines in defs.h? e.g.
```
ICOL_WHITE = (PAL16_YELLOW+5),
ICOL_BLUE = (PAL16_BLUE+5),
ICOL_RED = (PAL16_RED+5),
```

Reply:
Good catch, added it.